external-crates/move/crates/move-compiler/src/diagnostics/mod.rs

```diff
@@ -103,11 +104,56 @@ enum MigrationChange {
 
 // All of the migration changes
 pub struct Migration {
-    files: SimpleFiles<Symbol, String>,
-    file_mapping: FileMapping,
+    mapped_files: MappedFiles,
     changes: BTreeMap<FileId, BTreeMap<usize, Vec<(usize, MigrationChange)>>>,
 }
 
+/// A mapping from file ids to file contents along with the mapping of filehash to fileID.
+pub struct MappedFiles {
```

Comment:
These names lol, MappedFiles contains FileMapping 😵‍💫

Reply:
Maybe time to get some more meaningful names is my point. This sort of self referential boot strapping makes it a bit hard to really know whats going on

Reply:
Agreed, it's actually easy after this change to remove the `FileMapping` alias completely and instead use `MappedFiles` in its place :)